packages/react/.stylelintrc

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": "stylelint-config-recommended-scss",
+  "rules": {
+    "declaration-property-unit-blacklist": [
+      {
+        "font-size": [
+          "em",
+          "px",
+          "pt"
+        ],
+        "margin": [
+          "px",
+          "rem"
+        ],
+        "padding": [
+          "px",
+          "rem"
+        ],
+        "transition": [
+          "s",
+          "ms"
+        ]
+      },
+      {
+        "severity": "error"
+      }
+    ],
+    "declaration-property-value-blacklist": [
+      {
+        "color": [
+          "/^#/",
+          "/^rgb/"
+        ]
+      },
+      {
+        "severity": "error"
+      }
+    ]
+  }
+}
```

Comment:
moved stylelint rules into it's own file for better integration with vscode extensions (they don't read it from package.json)